workloads/templates/workload-nodevertical-script-cm.yml.j2

```diff
@@ -143,3 +168,121 @@ data:
           privileged: false
       nodeSelector:
         nodevertical: 'true'
+  conprof.yaml.template: |
+    scrape_configs:
+    - job_name: 'apiserver0'
+      scrape_interval: 30s
+      scrape_timeout: 10m
+      scheme: https
+      tls_config:
+        insecure_skip_verify: true
+      static_configs:
+      - targets: ['apiserver0-openshift-kube-apiserver.apps.{{clustername}}.perf-testing.devcluster.openshift.com']
+      bearer_token: {{bearer_token.stdout}}
+      profiling_config:
+        pprof_config:
+          heap:
+            enabled: true
+          profile:
+            enabled: true
+          goroutine:
+            enabled: false
+          threadcreate:
+            enabled: false
+          allocs:
+            enabled: false
+          block:
+            enabled: false
+          cmdline:
+            enabled: false
+          mutex:
+            enabled: false
+          trace:
+            enabled: false
+    - job_name: 'apiserver1'
+      scrape_interval: 30s
+      scrape_timeout: 10m
+      scheme: https
+      tls_config:
+        insecure_skip_verify: true
+      static_configs:
+      - targets: ['apiserver1-openshift-kube-apiserver.apps.{{clustername}}.perf-testing.devcluster.openshift.com']
```

Comment:
Do we have to modify the target when using a different base domain instead of perf-testing.devcluster.openshift.com or it doesn't matter?